etl/steps/data/grapher/happiness/2023-03-20/happiness.meta.yml

```diff
@@ -0,0 +1,171 @@
+dataset:
```

Comment:
@pabloarosado can I please ask you to check the **structure** of this migrated data page YAML before I delegate it to their owners? I'm looking for structural issues that I could correct in all migrated data pages, not for specific metadata issues (these will be up to owners to fix).

Here's [ETL-based data page](http://staging-site-mojmir/admin/datapage-preview/577009) and here's [JSON based](https://ourworldindata.org/grapher/happiness-cantril-ladder). Thank you!

Reply:
This one looks good, do you want me to check the others?

Reply:
Nope, thanks a lot!